pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -508,7 +508,7 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 	volumePluginName := unknownVolumePlugin
 	volumePlugin, err :=
 		og.volumePluginMgr.FindPluginBySpec(volumeToMount.VolumeSpec)
-	if err == nil && volumePlugin != nil {
+	if err == nil {
```

Comment:
> Do you see this check causing a problem? 

No

> I think it's better for the error handling to be more resilient in case the function changes its return values in the future.

A consensus is reached on the point that `volumePlugin` will be nil only when an error is returned. So i think this nil comparison is unnecessary.